### PR TITLE
Various fixes to README.cmake.md

### PR DESCRIPTION
**Pull request by ivancich**
Since using cmake is now being preferred, this is an attempt to correct a mistake and beef up the documentation. @alimaredia, this might be a good opportunity for you to further document the various cmake options you've implemented as well. @cbodley: is there anything else you'd like to see here?

### CODE REVIEW

**Comment by alimaredia** [comment]
@tchaikov @liewegas pretty minor changes but just wanted to give you a heads up on them.

**Comment by liewegas** [comment]
Umm.. can we remove README.cmake.md and put any net new content into README.md instead?

File: README.cmake.md [hunk 1]
```diff
@@ -103,8 +109,8 @@ Development
 
 The **run-cmake-check.sh** script will install Ceph dependencies,
 compile everything in debug mode, and run a number of tests to verify
-that the result behaves as expected. It will also build in-source
-(i.e., create a *build* directory).
+that the results behavek as expected. It will also build in-source
+(i.e., create a *build* directory within the Ceph repo's directory).
 
     $ ./run-cmake-check.sh
```
Comment (liewegas) [inline README.cmake.md]: this will go away shortly, and run-make-check.sh will use cmake.

**Comment by alimaredia** [comment]
@liewegas wow I thought this was a PR for README.md, good call,  I'll get to work on combining those two. @ivancich care to beef up README.md?

**Pull request closed without merging**